source/code/scxcorelib/pal/scxcondition.cpp

```diff
@@ -94,6 +98,9 @@ namespace SCXCoreLib
         {
             throw SCXErrnoException(L"pthread_mutex_destroy() function call failed", err, SCXSRCLOCATION);
```

Comment:
Given that we do throws all over the place, I'm REALLY curious what makes this throw different than the other throws:

```
jeffcof:code> grep -R throw | grep -v \\throws | grep -v // | wc -l
685
jeffcof:code> 
```

So not counting SCX or other providers, JUST the PAL, what makes this throw different from 684 other throws?

Please research this, I'd prefer fixing the code than masking the error if we can. Otherwise looks fine.

Will mark awaiting feedback to see what you determine here.